### DISCUSSION TITLE
React leaft in next-js

### DISCUSSION THREAD

**Discussion by nelsonmandeladev** [vercel/next.js#52714] [Help]
### Summary

Hello there, I'm working on simple next-js that I will like to add **react-leaflet** for map support, I'm wondering what is the best way of achieving this more efficiently.

Any suggestion will be welcomed, thanks.

**Comment by mintxenva75** [comment]
You are Building  an App which Suggests you would like to have  an Next.js leaflet for your Project that you are building is just an add Component you  can write this piece of code to add via your CLI Terminal ![Screenshot_20230715-112325.png](https://github.com/vercel/next.js/assets/99867219/6cb802dd-41a9-4680-9dd8-7ecc494b8efd) if you like to try an example from FreeCodeCamp and Enjoy your React js App

**Reply by nelsonmandeladev** [reply]
@mintxenva75 great, 
Let me try this and get back to you,
Thank's.

**Reply by mintxenva75** [reply]
Your welcome happy coding hope all is successfully written and no error bugs of course please share your results here i am sure it would be an breeze to compile .
